### ISSUE TITLE
v3.2.0 fails to install due to missing file

### ISSUE THREAD

**Issue by gitschwifty** [lingui/js-lingui#859]
**Describe the bug**
The 3.2.0 version of Lingui cli fails to install due to file `@lingui/cli/lingui.js` missing.

**To Reproduce**

```
npm i -D @lingui/cli@latest
```

**Expected behavior**
Lingui installs into node_modules

**Additional context**
Tested with 3.1.0 and 2.9.2 (currently using in another project so known good) installs to make sure it was not due to a local issue.

Error message with personal data removed:
```
npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path .../node_modules/@lingui/cli/lingui.js
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '.../node_modules/@lingui/cli/lingui.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 
```

**Comment by ifiokjr** [comment]
It looks like the project published without building: 

- `@lingui/cli@3.2.0` https://unpkg.com/browse/@lingui/cli@3.2.0/
- `@lingui/cli@3.1.0` https://unpkg.com/browse/@lingui/cli@3.1.0/

This seems to be the case for all the scoped packages.

**Comment by semoal** [comment]
Was the first time releasing with lerna, and we found a minor issue with contents --dir, already released a patch 3.2.1, and we'll work to fix the automatic versioning!

**Comment by tricoder42** [comment]
Sorry everyone. We're trying to automate our release pipeline and apparently there's a glitch.

I've just published [3.2.1](https://github.com/lingui/js-lingui/releases/tag/v3.2.1) and tested it locally.
